### DISCUSSION TITLE
HW3 Part 2

### DISCUSSION THREAD

**Discussion by awc17002** [cooperrc/computational-mechanics#115] [General]
Simple question: For part 2 of the hw, is there a data file for a lacrosse ball? Or should I use the tennis ball data with the radius and mass changed to reflect the mass and radius of a lacrosse ball?

**Comment by m-c-johnston** [comment]
Don't think so, I just used the mass and radius provided for the lacrosse ball and stuffed them into the fall drag function.

**Reply by poutrej** [reply]
My graph doesn't change when plugging in new values. Intuitively I feel like it should change because of drag changes

**Reply by m-c-johnston** [reply]
I did a difference plot and there was a very small change between the two falls when looked at like that. Otherwise the graphs did look identical.
There was a piece in one of the readings that showed a similar instance but don't remember where it was atm.

**Reply by OwenFuhr** [reply]
I changed the code to start the drop from a higher initial height and the difference was more pronounced.
